Collections/Collections.pier

```diff
@@ -91,36 +91,36 @@ optimized or occasionally even forbidden by subclasses of ==Collection==.
 | ==converting==	|	==asBag==, ==asSet==, ==asOrderedCollection==, ==asSortedCollection==, ==asArray==, ==asSortedCollection:==
 | ==creation== | ==with:==, ==with:with:==, ==with:with:with:==, ==with:with:with:with:==, ==withAll:==
 
+
 Beyond this basic uniformity, there are many different kinds of collection
 either supporting different protocols, or providing different behaviour for the
 same requests. Let us briefly survey some of the key differences:
 
-;==Sequenceable:==
+;""Sequenceable:""
```

Comment:
I'm not sure the edits on lines 90-112 of `Collections.pier` are necessary (specifically, changing the labels from mono-font (`==Collections:==`) to explicitly bold (`""Collections:""). Since this is a definition list, the terms will be bolded anyway, no sense in also marking them as bold.